### DISCUSSION TITLE
WOW + Windows Protools 12.5 success

### DISCUSSION THREAD

**Discussion by Bearsound** [apohl79/audiogridder#791] [Ideas]
I dreamed years ago on the posibility on this software became a reality. Teleport and Wormhole come to mind easily to veteran users. Thanks for develop it! And thanks for making it Open Source.

After testing it deeply I realize sadly that it doesnt work on PT 10 Mac (favorite music platform). Not even with TransVST nor BlueCat Patch.  

I were able to run the beta plug on Protools 12 but I had to manually make a folder with the name+ X64 folder and at last dropping the file. 

I mean, the installer isnt flawless, at least on the last beta.

What are the dependencies for audiogridder?  Do it really need to be a AAX **64**  bit? Or Intel even? It would be more universally used if AAX 32 bit **is** included (Protools 10 in mind) or a RTAS version of the plug is included. That could open its use (for me at least) a lot. 

I dream to be able to use PowerPC VST  or AU on a modern DAW. Even with no programmer skill I am tempted to do a fork focused on PowerPC. I made this post on MacRumors [https://forums.macrumors.com/threads/audiogridder-powerpc.2321977/](url) asking coders some help. 

Count with me as a beta tester for AAX32

**Comment by custa1200** [comment]
Wanting anybody to support PowerPC anything is ludicrous.  Apple have moved from it to Intel like 15 years ago and are now making the move to their own silicon.
